Assignment4/src/main/java/com/example/Assignment4/config/AsynchronousConfig.java

```diff
@@ -0,0 +1,29 @@
+package com.example.Assignment4.config;
+
+import org.springframework.beans.factory.annotation.Configurable;
+import org.springframework.context.annotation.Bean;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+import java.util.concurrent.Executor;
+
+/*
+ * configuration class
+ */
+@Configurable
+@EnableAsync
+public class AsynchronousConfig {
+    /*
+     * Defining the execution of thread using complete future
+     */
+    @Bean(name = "taskExecutor")
+    public Executor taskExecutor() {
+        var executor = new ThreadPoolTaskExecutor();
```

Comment:
-  final is missing
-  Constant should coming from yaml as it is a config which can change according to env

Reply:
done